### PR TITLE
Cosmetic changes + fixing comments according to Galois review.

### PR DESCRIPTION
**Pull request by drucker-nir**
1) Cosmetic changes (merging from the latest BIKE code)
2) Fixing comments according to Galois review.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pq-crypto/bike/defs.h [hunk 1]
```diff
@@ -1,115 +1,120 @@
...
+* The license is detailed in the file LICENSE.txt, and applies to this file.
+* ***************************************************************************/
+
+#pragma once
+
+////////////////////////////////////////////
+//             Basic defs
+///////////////////////////////////////////
+
+#ifdef __cplusplus
```
Comment (andrewhop) [inline pq-crypto/bike/defs.h:18]: Whats the motivation behind this? Is the upstream BIKE being updated to support c/c++?

File: pq-crypto/bike/bike1_l1_kem.c [hunk 2]
```diff
@@ -1,509 +1,488 @@
...
+    pk_t *l_pk = (pk_t *)pk;
+    
+    status_t res = SUCCESS;
+
+    // For DRBG and AES_PRF
+    double_seed_t seeds = {0};
+    aes_ctr_prf_state_t h_prf_state = {0};
+    
+    // Get the entrophy seeds
+    get_seeds(&seeds, KEYGEN_SEEDS);
```
Comment (andrewhop) [inline pq-crypto/bike/bike1_l1_kem.c]: Why remove these safety checks?

File: pq-crypto/bike/bike1_l1_kem.h [hunk 3]
```diff
@@ -12,8 +12,6 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-#ifndef __BIKE1_L1_KEM_H_INCLUDED__
-#define __BIKE1_L1_KEM_H_INCLUDED__
```
Comment (andrewhop) [inline pq-crypto/bike/bike1_l1_kem.h:15]: Pragma once?

File: pq-crypto/bike/defs.h (hunk 1 above)
Comment (drucker-nir) [reply pq-crypto/bike/defs.h:18]: Upstream BIKE can run with NTL, a C++ library.

File: pq-crypto/bike/bike1_l1_kem.h (hunk 3 above)
Comment (drucker-nir) [reply pq-crypto/bike/bike1_l1_kem.h:15]: Agree

**Review by andrewhop — APPROVED** [review]

**Pull request merged**